config.go

```diff
@@ -318,6 +340,29 @@ func (c *compiledModule) ExportedMemories() map[string]api.MemoryDefinition {
 	return c.module.ExportedMemories()
 }
 
+// CustomSections implements CompiledModule.CustomSections
+func (c *compiledModule) CustomSections() []api.CustomSection {
+	var ret []api.CustomSection
+	for _, d := range c.module.CustomSections {
+		dd := &customSection{data: d.Data, name: d.Name}
+		ret = append(ret, dd)
+	}
+	return ret
+}
+
+type customSection struct {
```

Comment:
```suggestion
// customSection implements ...
type customSection struct {
```

Reply:
BTW is it fine to implement it here or should it go some place else?

Reply:
i think for now it's fine to implement here (in any way we can change it later when necessary because it's not exported!)